source/isaaclab/isaaclab/sensors/ray_caster/multi_mesh_ray_caster_cfg.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+"""Configuration for the ray-cast sensor."""
+
+
+from dataclasses import MISSING
+
+from isaaclab.utils import configclass
+
+from .multi_mesh_ray_caster import MultiMeshRayCaster
+from .ray_caster_cfg import RayCasterCfg
+
+
+@configclass
+class MultiMeshRayCasterCfg(RayCasterCfg):
+    """Configuration for the multi-mesh ray-cast sensor."""
+
+    @configclass
+    class RaycastTargetCfg:
+        """Configuration for different ray-cast targets."""
+
+        target_prim_expr: str = MISSING
+        """The regex to specify the target prim to ray cast against."""
+
+        is_global: bool = False
+        """Whether the target prim is a global object or exists for each environment instance. Defaults to False."""
```

Comment:
As I was writing more explanation, I was wondering if the expression resolves to only 1 prim, is it safe to assume that it is a global target? Otherwise, let's say, user defines this attribute to `False` but passes "/World/ground". Would in this case, this only be used to the first env?

```suggestion
        """Whether the target prim is a global object or exists for each environment instance. Defaults to False.
        
        If True, the target prim is used for ray-casting in all environments.
        Otherwise, it is only used per environment.
        """
```

Reply:
Very good point! I think right now, it just fails tbh.